sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernateIndexDao.java

```diff
@@ -65,24 +76,51 @@ public IndexFamily getIndexFamilyByName(String name) {
   }
 
   @Override
-  public List<Index> listAllIndices(PlatformType platformType) {
-    Query query = currentSession().createQuery("from Index where family.platformType = :platform");
-    query.setParameter("platform", platformType);
-    @SuppressWarnings("unchecked")
-    List<Index> list = query.list();
-    return list;
+  public String getProjectColumn() {
+    return null;
```

Comment:
`HibernatePaginatedDataSource#restrictPaginationByProjectId` will try adding a restriction to the criteria based on this. Should that method check for null and report to errorHandler instead?